### DISCUSSION TITLE
Any recommendations for RSS on Desktop

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [spacecowboy/Feeder#215] [Q&A] [answered]
The `sync` in feeder only works with  feeder-app on other devices. 

So is `Waydroid` the way to go?

Do you use rss-feeds on desktop?

**Comment by spacecowboy** [comment, marked as answer]
I only read RSS in Feeder, which is why it has no sync support for other services.

I have no recommendations to make since I don't use anything else.

**Comment by githubtefo** [comment]
Would be nice to have any sync way with any RSS reader for Desktop.
The truth is I can't find any Desktop Apps as good as Feeder for mobile.

**Reply by KAGEYAM4** [reply]
I found these, but sync is the problem->
https://flathub.org/apps/app.feeddeck.feeddeck
https://flathub.org/apps/me.hyliu.fluentreader
https://flathub.org/apps/io.gitlab.news_flash.NewsFlash

**Reply by spacecowboy** [reply]
indeed. there exists no standard protocol.

**Reply by DraconicNEO** [reply]
Maybe at some point Feeder could get a desktop version. Then we'd get all the benefits of the mobile version (including sync) but also on Desktop. Would probably require a lot of work though to convert the UI to one more desktop friendly, plus I never converted android apps to desktop and I'm willing to bet it isn't as easy as simply taking the java code from this app and setting it up as a desktop app.

**Comment by prakhar-garg-dev** [comment]
I tried RSSGaurd although ui not that material gives features

**Reply by DraconicNEO** [reply]
RSSGuard handles certain feeds badly though, I transferred my feeds and 3 of them wouldn't load up, either due to a blank space at the top or due to some other weird issue I couldn't identify (wouldn't give an error, feed was just empty).

**Reply by prakhar-garg-dev** [reply]
There are several others but there are some flaws in each one of them hope so some others will come soon which are more comprehensive
